apps/stats-web/src/app/addresses/[address]/deployments/[dseq]/page.tsx

```diff
@@ -39,8 +48,12 @@ async function fetchDeploymentData(address: string, dseq: string, network: strin
   return response.json();
 }
 
-export default async function DeploymentDetailPage({ params: { address, dseq }, searchParams: { network } }: IProps) {
-  const deployment = await fetchDeploymentData(address, dseq, network as string);
+export default async function DeploymentDetailPage(props: DeploymentDetailPageProps) {
+  const {
+    params: { address, dseq },
+    searchParams: { network }
+  } = DeploymentDetailPageSchema.parse(props);
```

Comment:
This will throw an error if the props are not correct?

Reply:
yea

Reply:
I think network id should have a default value of mainnet.

Reply:
so let's imagine someone implements a link to this page on the frontend. The one may notice that it works without the network params and not bother why it's needed. Thus we may get a bug when a mainnet link is opened from, say, staging.
I'd say it should fail in dev.

Reply:
I suppose yea, we do append the network param to the path everywhere is use it so it should be fine.